frame/scheduler/src/lib.rs

```diff
@@ -161,24 +188,48 @@ decl_module! {
 		}
 
 		/// Cancel a named scheduled task.
-		#[weight = 100_000]
+		///
+		/// # <weight>
+		/// - S = Number of already scheduled calls
+		/// - Base Weight: 24.91 + 2.907 * S µs
+		/// - DB Weight:
+		///     - Read: Agenda, Lookup
+		///     - Write: Agenda, Lookup
+		/// - Will use base weight of 100 which should be good for up to 30 scheduled calls
+		/// # </weight>
+		#[weight = 100_000_000 + T::DbWeight::get().reads_writes(2, 2)]
 		fn cancel_named(origin, id: Vec<u8>) {
 			ensure_root(origin)?;
 			Self::do_cancel_named(id)?;
 		}
 
+		/// Execute the scheduled calls
+		///
+		/// # <weight>
+		/// - S = Number of already scheduled calls
+		/// - Base Weight: 9.243 + 23.45 * S µs
+		/// - DB Weight:
+		///     - Read: Agenda, Lookup
+		///     - Write: Agenda, Lookup
+		/// - In this case, the base weight is dominated by read/write operations.
+		///   To simplify, we will only account for additional reads/writes.
```

Comment:
what does that mean is `23.45 * S` negligible ?

Reply:
added it back